hwdb.d/70-mouse.hwdb

```diff
@@ -41,6 +41,7 @@
 #   udevadm info /dev/input/eventXX.
 #
 # Allowed properties are:
+#   ID_INPUT_3D_MOUSE
```

Comment:
what is the device tagged as atm? Is it already `ID_INPUT_MOUSE` (and handled by libinput as such?) or does it have any `ID_INPUT` tags at all? If we're defining a new attribute we might as well define our expectations like we did for `ID_INPUT_TRACKBALL` (see line 53+).

Reply:
Shows up as a joystick with 6-axis:
```
P: /devices/virtual/misc/uhid/0005:256F:C63A.0012
M: 0005:256F:C63A.0012
R: 0012
U: hid
V: hid-generic
E: DEVPATH=/devices/virtual/misc/uhid/0005:256F:C63A.0012
E: SUBSYSTEM=hid
E: DRIVER=hid-generic
E: HID_ID=0005:0000256F:0000C63A
E: HID_NAME=SpaceMouse Wireless BT
E: HID_PHYS=3c:e9:f7:60:d8:48
E: HID_UNIQ=cb:a6:c9:1b:6f:7d
E: MODALIAS=hid:b0005g0001v0000256Fp0000C63A

P: /devices/virtual/misc/uhid/0005:256F:C63A.0012/hidraw/hidraw6
M: hidraw6
R: 6
U: hidraw
D: c 241:6
N: hidraw6
L: 0
E: DEVPATH=/devices/virtual/misc/uhid/0005:256F:C63A.0012/hidraw/hidraw6
E: SUBSYSTEM=hidraw
E: DEVNAME=/dev/hidraw6
E: MAJOR=241
E: MINOR=6

P: /devices/virtual/misc/uhid/0005:256F:C63A.0012/input/input38
M: input38
R: 38
U: input
E: DEVPATH=/devices/virtual/misc/uhid/0005:256F:C63A.0012/input/input38
E: SUBSYSTEM=input
E: PRODUCT=5/256f/c63a/504
E: NAME="SpaceMouse Wireless BT"
E: PHYS="3c:e9:f7:60:d8:48"
E: UNIQ="cb:a6:c9:1b:6f:7d"
E: PROP=0
E: EV=2001b
E: KEY=3 0 0 0 0
E: ABS=3f
E: MSC=10
E: LED=100
E: MODALIAS=input:b0005v256FpC63Ae0504-e0,1,3,4,11,k100,101,ra0,1,2,3,4,5,m4,l8,sfw
E: USEC_INITIALIZED=289995105945
E: ID_INPUT=1
E: ID_INPUT_JOYSTICK=1
E: ID_BUS=bluetooth
E: TAGS=:seat:
E: CURRENT_TAGS=:seat:

P: /devices/virtual/misc/uhid/0005:256F:C63A.0012/input/input38/event11
M: event11
R: 11
U: input
D: c 13:75
N: input/event11
L: 0
E: DEVPATH=/devices/virtual/misc/uhid/0005:256F:C63A.0012/input/input38/event11
E: SUBSYSTEM=input
E: DEVNAME=/dev/input/event11
E: MAJOR=13
E: MINOR=75
E: USEC_INITIALIZED=289995106217
E: ID_INPUT=1
E: ID_INPUT_JOYSTICK=1
E: ID_INPUT_WIDTH_MM=2
E: ID_INPUT_HEIGHT_MM=2
E: ID_BUS=bluetooth
E: ID_INPUT_JOYSTICK_INTEGRATION=external
E: LIBINPUT_DEVICE_GROUP=5/256f/c63a:3c:e9:f7:60:d8:48
E: TAGS=:seat:uaccess:
E: CURRENT_TAGS=:seat:uaccess:
```